akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -754,7 +754,7 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
 
   def receiveGossipStatus(status: GossipStatus): Unit = {
     val from = status.from
-    if (!latestGossip.overview.reachability.isReachable(selfUniqueAddress, from))
+    if (!latestGossip.isReachable(selfUniqueAddress, from))
```

Comment:
yes, we are on the same page. I didn't know what `isReachable` did when writing this comment. inter-team observations should "always" be ignored.